### ISSUE TITLE
About multi-GPU training learning rate

### ISSUE THREAD

**Issue by RayYoh** [filaPro/oneformer3d#78]
Hi @filaPro, recently, I have been trying to reproduce the results of Oneformer3D to compare with my own method. Since using just one GPU would be relatively slow, I changed to using multi-GPU training to align the configs with my method.  
My question is if I use 3 or 4 GPUs for training, which means the batch size will be 3*4 or 4*4, do I need to modify the learning rate in the raw configs (i.e. 3*10-4 or 4*10-4).

Best

**Comment by filaPro** [comment]
We never tried multi-gpu for oneforner3d, so probably yes, multiplying should be fine, but i'm not sure if after it the metrics will be reproduced exactly.
